docs/install/any-kubernetes-cluster.md

```diff
@@ -23,15 +23,20 @@ showlandingtoc: "false"
 You can install Knative by applying YAML files using the `kubectl` CLI.
 You can install the Serving component, Eventing component, or both on your cluster.
 
+## System Minimum Requirements
+For prototyping purposes, Knative can work on local deployments of Kubernetes (minikube, kind, etc.).
```

Comment:
```suggestion
For prototyping purposes, Knative can work on local deployments of Kubernetes.
```
Wouldn't mention these by name because people then start asking for specific docs for them and we don't do per-platform docs.

Reply:
I think it's important to mention them by name... when people ask we will say "no"

Reply:
Also, this is a bit confusing now I think because we don't give min requirements for the local deployments, just the production deployments? Should we even be saying production purposes before we're calling everything v 1.0? cc @n3wscott @evankanderson

Reply:
I don't mind removing these bits altogether, my core concern is not having system requirements that are beyond people's local machines at the top of the Install doc (I don't want to scare people)

Reply:
What's the point not telling them the truth though? If the requirements are beyond the scope of what they have access to we can't really do anything about that, why would we want someone to start using it anyway if their machine can't handle it and eventually it just won't work and will have wasted their time? These need to be verified properly anyway but that to me would be better, i.e, make them accurate instead of just removing them.

Also, I disagree about it being important to mention minikube and kind, it's just going to annoy people if you tell them they can use those but that we're not going to produce docs for it, and it mixes the message we were trying to send by removing platform specific docs in the first place IMO.

Reply:
My only concern is making sure people know that a prototype of Knative will work on their local machine... 

>If the requirements are beyond the scope of what they have access to we can't really do anything about that, why would we want someone to start using it anyway if their machine can't handle it and eventually it just won't work and will have wasted their time?

We aren't saying "you can run a production application with the resources of your laptop" but we do want to say "hey you can try this out with basically any K8s deployment." 

>Also, I disagree about it being important to mention minikube and kind, it's just going to annoy people if you tell them they can use those but that we're not going to produce docs for it, and it mixes the message we were trying to send by removing platform specific docs in the first place IMO.

I yield on this point 👍🏼